specification/containerinstance/resource-manager/Microsoft.ContainerInstance/stable/2018-10-01/containerInstance.json

```diff
@@ -544,6 +544,68 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ContainerInstance/locations/{location}/cachedImages": {
+      "get": {
+        "operationId": "ListCachedImages",
+        "x-ms-examples": {
+          "CachedImages": {
+            "$ref": "./examples/CachedImagesList.json"
+          }
+        },
+        "summary": "Get the list of cached images.",
+        "description": "Get the list of cached images on specific OS type for a subscription in a region.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/LocationParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
Please add the "default" response to catch all the error states. Sample in batch RP swagger.